src/Row/Row.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from 'react'
+import styled, { css } from 'styled-components'
+
+import { theme } from '../theme'
+import { Icon } from '../Icon'
+import { Box } from '../Box'
+import { Text } from '../Text'
+
+export type Props = {
+  icon?: string
+  heading: string
+  subHeading?: string
+  type?: 'first' | 'last' | 'middle' | 'curved'
+  caret?: boolean
+}
+
+export const Row: FC<Props> = ({
+  icon,
+  heading,
+  subHeading,
+  type,
+  caret = true,
+}) => {
+  return (
+    <Container type={type}>
+      <Box>{icon && <Icon render={icon} size={24} />}</Box>
```

Comment:
I'm kinda wondering if instead of having a fixed grid-template and render empty divs to make them fill the 24px space of the template we should probably have dynamic grid-templates based on icon and caret and just do something like:
```
{icon && <Icon render={icon} size={24} />}
...
{caret && <Icon render="caret" rotate={-90} color="grey8" />}
```

What do you think?